integrations.yml

```diff
@@ -185,3 +185,9 @@
   website: "https://www.getharvest.com/basecamp-time-tracking"
   icon: "harvest_logo.png"
   category: 2
+ -
```

Comment:
The `-` should be at the beginning of the line. Need to remove the leading space.